packages/teams-js/test/private/legacy.spec.ts

```diff
@@ -28,15 +28,17 @@ describe('AppSDK-privateAPIs', () => {
   });
   describe('getUserJoinedTeams', () => {
```

Comment:
Can we add unit tests for isSupported here?

Reply:
Good catch!

Reply:
Added unit test case for isSupported in the latest commit.